### ISSUE TITLE
Reactive Tazer during Winter Wyvern ult

### ISSUE THREAD

**Issue by Aquintus** [jeffhill/Dota2#725]
### Ability name

Reactive Tazer

### Description

Not sure if intended or not but Techies "Reactive Tazer" does not work on teammates while being the target of a Winter Wyvern ultimate.

All good if it's intended that way, otherwise it could be designed to be a sort of counter to Wyvern ult as Techies can cast it on allies with his aghanim scepter.
![tazer1](https://user-images.githubusercontent.com/40473897/172573430-3cbf8a84-f503-43d2-ac26-f30acd22d9eb.png)
![tazer2](https://user-images.githubusercontent.com/40473897/172573468-c256c189-71bb-463b-8bdd-6033598b9591.png)
![tazer3](https://user-images.githubusercontent.com/40473897/172573497-aabdc7bf-0fc7-4abe-afec-ead03e455dc1.png)

### Example Match ID (and possibly Timestamp)

6606511777 - 23:58

**Comment by arhen** [comment]
> Enemies attacking Techies are temporarily disarmed.

I think it's because ur teammates is not enemy?
If that desc goes to "unit" instead enemy then it will work I  think.

**Comment by Aquintus** [comment]
Yeah the description covers most cases since allies usually don't attack you. 

Was just throwing it out there since it happened yesterday and it's questionable logically how a tazer can differentiate as well as being a potential mechanical counter if you throw in the aghanim scepter of techies when playing vs. a WW. I am fine either way. :D

**Comment by pseudoparenchymatous** [comment]
Intended behaviour, not a bug. You can close this
